### PR TITLE
Feature/dx 255 invoice section

### PR DESCRIPTION
**Pull request by NullableInt**
Updated invoice section to be more include inclusive and fix links

### CODE REVIEW

File: _includes/callback-reference.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+and the two `GET` requests that you make to get the updated status.
+
+```mermaid
+sequenceDiagram
+  Participant Merchant
+  Participant PAYEX
+
+  activate Merchant
+  activate PAYEX
+  PAYEX->Merchant: POST <callbackUrl>
```
Comment (trieutran91) [inline _includes/callback-reference.md]:
```suggestion
  PAYEX->>Merchant: POST <callbackUrl>
```

Double "greater than" gives arrowhead

File: _includes/callback-reference.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  note left of Merchant: Callback by PayEx
+  Merchant-->PAYEX: HTTP response
```
Comment (trieutran91) [inline _includes/callback-reference.md]:
```suggestion
  Merchant-->>PAYEX: HTTP response
```
Double "greater than" gives arrowhead

File: _includes/callback-reference.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  deactivate PAYEX
+  deactivate Merchant
+
+  activate Merchant
+  Merchant->PAYEX: GET <payment instrument> payment
```
Comment (trieutran91) [inline _includes/callback-reference.md]:
```suggestion
  Merchant->>PAYEX: GET <payment instrument> payment
```

Double "greater than" gives arrowhead

File: _includes/callback-reference.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  note left of Merchant: First API request
+  activate PAYEX
+  PAYEX-->Merchant: payment resource
```
Comment (trieutran91) [inline _includes/callback-reference.md]:
```suggestion
  PAYEX-->>Merchant: payment resource
```
Double "greater than" gives arrowhead

File: _includes/operations-reference.md [hunk 5]
```diff
@@ -0,0 +1,511 @@
...
+            "href": "https://api.externalintegration.payex.com/psp/paymentorders/b80be381-b572-4f1e-9691-08d5dd095bc4/reversals",
+            "rel": "create-paymentorder-reversal",
+            "contentType": "application/json"
+        }
+    ]
+}
+```
+
+{:.table .table-striped}
+| **Property**| **Description**
```
Comment (trieutran91) [inline _includes/operations-reference.md]:
```suggestion
| **Property**| **Description**
|:--------------|:------------------
```
Add header separator

File: _includes/operations-reference.md [hunk 6]
```diff
@@ -0,0 +1,511 @@
...
+described here in the documentation. 
+Always use the `href` and `method` as specified in the response by finding 
+the appropriate operation based on its `rel` value. 
+The only thing that should be hard coded in the client is the value of 
+the `rel` and the request that will be sent in the HTTP body of the request 
+for the given operation.
+
+
+{:.table .table-striped}
+| **Operation** | **Description**
```
Comment (trieutran91) [inline _includes/operations-reference.md]:
```suggestion
| **Operation** | **Description**
|:----------------|:----------------
```
Add header separator

File: _includes/operations-reference.md [hunk 7]
```diff
@@ -0,0 +1,511 @@
...
+                "amount": 500,
+                "vatAmount": 125
+            }
+        ]
+    }
+}
+```
+
+{:.table .table-striped}
+| **Required** | **Property**| **Type**| **Description**
```
Comment (trieutran91) [inline _includes/operations-reference.md]:
```suggestion
| **Required** | **Property**| **Type**| **Description**
|:---------------|:-------------|:----------|:-----------------
```
Add header separator

File: _includes/operations-reference.md [hunk 8]
```diff
@@ -0,0 +1,511 @@
...
+            "vatAmount": 3122,
+            "description": "Capturing the authorized payment",
+            "payeeReference": "AB832",
+        }
+    }
+}
+```
+
+{:.table .table-striped}
+| **Property**| **Data type** | **Description**
```
Comment (trieutran91) [inline _includes/operations-reference.md]:
```suggestion
| **Property**| **Data type** | **Description**
|:--------------|:----------------|:-----------------
```

File: _includes/operations-reference.md [hunk 9]
```diff
@@ -0,0 +1,511 @@
...
+{
+    "transaction": {
+        "payeeReference": "ABC123",
+        "description": "Cancelling parts of the total amount"
+    }
+}
+```
+
+{:.table .table-striped}
+| **Property**| **Type**| **Required** | **Description**
```
Comment (trieutran91) [inline _includes/operations-reference.md]:
```suggestion
| **Property**| **Type**| **Required** | **Description**
|:-------------|:----------|:---------------|:------------------
```
Add head separator

File: _includes/operations-reference.md [hunk 10]
```diff
@@ -0,0 +1,511 @@
...
+            "vatAmount": 1122,
+            "description": "Cancelling parts of the authorized payment",
+            "payeeReference": "AB832",
+        }
+    }
+}
+```
+
+{:.table .table-striped}
+| **Property**| **Data type** | **Description**
```
Comment (trieutran91) [inline _includes/operations-reference.md]:
```suggestion
| **Property**| **Data type** | **Description**
|:--------------|:---------------|:---------------
```
Add head separator

File: _includes/operations-reference.md [hunk 11]
```diff
@@ -0,0 +1,511 @@
...
+    "transaction": {
+        "amount": 15610,
+        "vatAmount": 3122,
+        "payeeReference": "ABC123",
+        "description": "description for transaction"
+    }
+}
+```
+{:.table .table-striped}
+| **Property**| **Type**| **Required** | **Description**
```
Comment (trieutran91) [inline _includes/operations-reference.md]:
```suggestion
| **Property**| **Type**| **Required** | **Description**
|:-------------|:----------|:---------------|:---------------
```
Add head separator

File: _includes/operations-reference.md [hunk 12]
```diff
@@ -0,0 +1,511 @@
...
+            "vatAmount": 1122,
+            "description": "Reversing the capture amount",
+            "payeeReference": "ABC987",
+        }
+    }
+}
+```
+
+{:.table .table-striped}
+| **Property**| **Data type** | **Description**
```
Comment (trieutran91) [inline _includes/operations-reference.md]:
```suggestion
| **Property**| **Data type** | **Description**
|:-------------|:----------------|:-----------------
```
Add head separator

File: _includes/transactions-reference.md [hunk 13]
```diff
@@ -0,0 +1,133 @@
...
+            "failedReason": "",
+            "isOperational": "TRUE|FALSE",
+            "operations": []
+        }]
+    }
+}
+```
+
+{:.table .table-striped}
+| **Property** | **Data type** | **Required**
```
Comment (trieutran91) [inline _includes/transactions-reference.md]:
```suggestion
| **Property** | **Data type** | **Required**
|:--------------|:----------------|:---------------
```
Add head separator

File: _includes/transactions-reference.md [hunk 14]
```diff
@@ -0,0 +1,133 @@
...
+        "payeeReference": "AH123456",
+        "failedReason": "",
+        "isOperational": true,
+        "operations": []
+    }
+}
+```
+
+{:.table .table-striped}
+| **Property** | **Data type** | **Description**
```
Comment (trieutran91) [inline _includes/transactions-reference.md]:
```suggestion
| **Property** | **Data type** | **Description**
|:--------------|:----------------|:-----------------
```
Add header separator

File: payments/invoice/after-payment.md [hunk 15]
```diff
@@ -168,31 +178,36 @@ Content-Type: application/json
 ```
 
 {:.table .table-striped}
-| Property | Data type | Description
-| payment | string | The relative URI of the payment this capture transaction resource belongs to.
-| capture.itemDescriptions.id | string | The relative URI of the item descriptions resource associated with this capture transaction resource.
-| capture.invoiceCopy | string | The relative URI of the downloadable invoice copy in PDF format.
-| capture.transaction | object | The object representation of the [transaction][technical-reference-transaction].
+| **Property** | **Data type** | **Description**
```
Comment (trieutran91) [inline payments/invoice/after-payment.md]:
```suggestion
| **Property** | **Data type** | **Description**
|:--------------|:----------------|:-----------------
```
Add header separator

File: payments/invoice/after-payment.md [hunk 16]
```diff
@@ -120,21 +129,22 @@ Content-Type: application/json
...
-| transaction.payeeReference | string(50) | ✔︎ | A **unique **reference max 50 characters set by the merchant system) - this must be unique for each operation! The `payeeReference` must follow the regex pattern `[\w]* (a-zA-Z0-9_)`.
-| transaction.description | string | ✔︎ | A textual description of the capture
-| itemDescriptions.amount | integer | ✔︎ | Total price for this order line - entered in the lowest momentary units of the selected currency. E.g. 10000 = 100.00 NOK, 5000 = 50.00 SEK.
-| itemDescriptions.description | string | ✔︎ | A textual description of this product
-| vatSummary.amount | integer | ✔︎ | Total price for this order line - entered in the lowest momentary units of the selected currency. E.g. 10000 = 100.00 NOK, 5000 = 50.00 SEK.
-| vatSummary.vatAmount | integer | ✔︎ | VAT Amount entered in the lowest momentary units of the selected currency. E.g. 10000 = 100.00 NOK, 5000 = 50.00 SEK.
-| vatSummary.vatPercent | string | ✔︎ | The VAT in percent. Supported values : "0.00", "6.00", "8.00", "10.00", "12.00", "14.00", "15.00", "22.00", "24.00", "25.00"
+| **Parameter name** | **Datatype** | **Required** | Value (with description)
```
Comment (trieutran91) [inline payments/invoice/after-payment.md]:
```suggestion
| **Parameter name** | **Datatype** | **Required** | Value (with description)
|:-----------------------|:---------------|:---------------|:-----------------------
```
Add header separator

File: payments/invoice/after-payment.md [hunk 17]
```diff
@@ -234,15 +249,16 @@ Content-Type: application/json
 ```
 
 {:.table .table-striped}
-| Property | Data type | Description
-| payment | string | The relative URI of the payment this list of cancellation transactions belong to.
-| cancellations.id | string | The relative URI of the current `cancellations` resource.
-| cancellations.cancellationList | array | The array of the cancellation transaction objects.
-| cancellations.cancellationList[] | object | The object representation of the cancellation transaction resource described below.
+| **Property** | **Data type** | **Description**
```
Comment (trieutran91) [inline payments/invoice/after-payment.md]:
```suggestion
| **Property** | **Data type** | **Description**
|:--------------|:----------------|:------------------
```
Add header separator

File: payments/invoice/after-payment.md [hunk 18]
```diff
@@ -263,12 +279,13 @@ Content-Type: application/json
 ```
 
 {:.table .table-striped}
-| Parameter name | Datatype | Required | Value (with description)
-| transaction.activity | string | ✔︎ | `FinancingConsumer`.
-| transaction.payeeReference | string | ✔︎ | A **unique **reference max 50 characters set by the merchant system) - this must be unique for each operation! The [payeeReference][technical-reference-payeeReference] must follow the regex pattern `[\w]* (a-zA-Z0-9_)`. 
-| transaction.description | string(50) | ✔︎ | A textual description for the cancellation.
+| **Required** | **Parameter name** | **Datatype** | **Value (with description)**
```
Comment (trieutran91) [inline payments/invoice/after-payment.md]:
```suggestion
| **Required** | **Parameter name** | **Datatype** | **Value (with description)**
|:--------------|:------------------------|:---------------|:----------------------
```
Add header separator

File: payments/invoice/after-payment.md [hunk 19]
```diff
@@ -297,29 +314,33 @@ The `cancel` resource will be returned, containing information about the newly c
 ```
 
 {:.table .table-striped}
-| Property | Data type | Description
-| payment | string | The relative URI of the payment this capture transaction belongs to.
-| reversal.id | string | The relative URI of the created capture transaction.
-| reversal.transaction | object | The object representation of the generic [transaction][technical-reference-transaction].
+| **Property** | **Data type** | **Description**
```
Comment (trieutran91) [inline payments/invoice/after-payment.md]:
```suggestion
| **Property** | **Data type** | **Description**
|:--------------|:----------------|:----------------
```
Add header separator

File: payments/invoice/after-payment.md [hunk 20]
```diff
@@ -361,10 +382,10 @@ Content-Type: application/json
 ```
 
 {:.table .table-striped}
-| Property | Type | Description
-| payment | string | The relative URI of the payment that the reversal transactions belong to.
-| reversalList | array | The array of reversal transaction objects.
-| reversalList[] | object | The reversal transaction object representation of the reversal transaction resource described below.
+| **Property** | **Type** | **Description**
```
Comment (trieutran91) [inline payments/invoice/after-payment.md]:
```suggestion
| **Property** | **Type** | **Description**
|:--------------|:-----------|:-----------------
```
Add header separator

File: payments/invoice/after-payment.md [hunk 21]
```diff
@@ -394,12 +415,12 @@ Content-Type: application/json
 **Properties**
 
 {:.table .table-striped}
-| Property | Data type | Required | Description
-| transaction.activity | string | Y| `FinancingConsumer`.
-| transaction.amount | integer | ✔︎ | Amount Entered in the lowest momentary units of the selected currency. E.g. *10000* = 100.00 NOK, *5000* = 50.00 SEK.
-| transaction.vatAmount | integer | ✔︎ | Amount Entered in the lowest momentary units of the selected currency. E.g. *10000* = 100.00 NOK, *5000* = 50.00 SEK.
-| transaction.payeeReference | string(50 | ✔︎ | A **unique **reference max 50 characters set by the merchant system) - this must be unique for each operation! The `payeeReference` must follow the regex pattern `[\w]* (a-zA-Z0-9_)`.
-| transaction.description | string | ✔︎ | A textual description of the reversal.
+| **Required** | **Property** | **Data type** | **Description**
```
Comment (trieutran91) [inline payments/invoice/after-payment.md]:
```suggestion
| **Required** | **Property** | **Data type** | **Description**
|:---------------|:--------------|:----------------|:----------------
```
Add header separator

File: payments/invoice/after-payment.md [hunk 22]
```diff
@@ -436,34 +457,36 @@ Content-Type: application/json
 **Properties**
 
 {:.table .table-striped}
-| Property | Data type | Description
-| payment | string | The relative URI of the payment this capture transaction belongs to.
-| reversal.id | string | The relative URI of the created capture transaction.
-| reversal.transaction | object | The object representation of the generic [transaction][technical-reference-transaction].
+| **Property** | **Data type** | **Description**
```
Comment (trieutran91) [inline payments/invoice/after-payment.md]:
```suggestion
| **Property** | **Data type** | **Description**
|:--------------|:----------------|:-----------
```
Add header separator

File: _includes/transactions-reference.md (hunk 13 above)
Comment (asbjornu) [inline _includes/transactions-reference.md]:
```suggestion
| Property | Data type | Required
|:--------------|:----------------|:---------------
```

**Pull request merged**